config/contour/external.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRoleBinding
 metadata:
-  name: contour-external
+  name: contour-external-crb
```

Comment:
It is better to rename `contour-external-knative` and `contour-internal-knative` for  `ClusterRoleBindings`? 
It will be consistent with ClusterRole name.